x-pack/platform/plugins/shared/ml/public/application/model_management/test_models/models/inference_base.ts

```diff
@@ -312,6 +314,8 @@ export abstract class InferenceBase<TInferResponse> {
         DEFAULT_INFERENCE_TIME_OUT
       )) as unknown as TRawInferResponse;
 
+      this.mlUsageCollection.count('tested_trained_model');
```

Comment:
Although it's good to have counts for when testing has been performed. I think it would be a lot more valuable to also track additional information, such as the model type or pytorch task type.

Reply:
I assume we need event-based telemetry for that